packages/sanity/src/desk/comments/src/components/pte/comment-input/CommentInput.tsx

```diff
@@ -169,13 +172,32 @@ export const CommentInput = forwardRef<CommentInputHandle, CommentInputProps>(
       [discardDialogController, requestFocus, resetEditorInstance, scrollToEditor],
     )
 
+    const handleFocus = useCallback(
```

Comment:
The `react-focus-lock` causes some issues with focus, especially when it comes to returning focus when e.g closing a popover with a comment input.

Reply:
A clarifying comment would be welcome here! (as `react-focus-lock` seems to be used just about everywhere else)

Reply:
It caused issues with returning focus + some focus issues when moving focus to the PTE